scripts/package-artifacts.ps1

```diff
@@ -22,23 +22,27 @@
     Compress-Archive -Path "$destination\*" -DestinationPath "$destination.zip" -Force
 }
 
-function Package-NetScanner([string]$sourcetfm, [string]$targettfm) {
```

Comment:
Nitpick: I would either rename this one to `Package-Net` or the previous function to `Package-NetFrameworkScanner`